### DISCUSSION TITLE
A plugin to center the dialog interface during cutscenes/when talking to npcs

### DISCUSSION THREAD

**Discussion by ShaunDreclin** [runelite/runelite#17298] [Ideas]
The new cutscene dialogue system is great but it's only used in the latest quests, a plugin to replicate the new system for old quests would be nice. Could optionally work for all npc dialogues as well, instead of just cutscenes

**Comment by broadcast** [comment]
I second this! I was just looking for something like this recently but haven't found anything. I'm not sure it would be a trivial change. Ideally there would be a way to "copy" the existing dialogue npc box to the center and hide the original rather than creating a custom reimplementation (if such a thing is even possible).
